platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -1043,7 +1042,6 @@ private void loadReconnectState(final SignedState signedState) {
 
         gossip.resetFallenBehind();
         eventCreator.resumeEventCreation();
-        platformStatusManager.submitStatusAction(new ReconnectCompleteAction(signedState.getRound()));
```

Comment:
I think it might have been better to just move this line up before the state saving is kicked off. That is a smaller change, and we also don't add a new dependency to reconnet

Reply:
Fixed